pyhap/characteristic.py

```diff
@@ -217,14 +232,16 @@ def to_HAP(self):
             HAP_REPR_FORMAT: self.properties[PROP_FORMAT],
         }
 
+        value = self.get_value()
+
```

Comment:
A small one: `value` is used in the block below, so why not delete the empty line?